test/sqllogictest/show.slt

```diff
@@ -168,12 +168,14 @@ statement error supported
 SHOW VIEWS WHERE "VIEWS" = 'u'
 
 query T
-SHOW SOURCES
+SHOW TABLES
```

Comment:
this test and the one below looks like it already exists on line 183 -- might be better to keep this as show sources and assert that the output is empty?

Reply:
Good idea. Done.